Makefile

```diff
@@ -174,6 +175,8 @@ up: checkout run/nginx/nginx.conf var/wp-data run/wp-nonces/wp-nonces.php run/ce
 	./devscripts/await-mariadb-ready
 	@echo "If you have want to use the wp-gutenberg-epfl plugin or to dev on Gutenberg,"
 	@echo "install nvm and run 'make gutenberg'"
+	@echo
+	firefox https://wordpress.localhost
```

Comment:
Please use xdg-open to open a URL here.

Reply:
Fixed in 9012192333367483018d1d6ff41d648e79b4e438